cmd/ping.go

```diff
@@ -36,4 +46,8 @@ can connect to Kong's Admin API or not.`,
 
 func init() {
 	rootCmd.AddCommand(pingCmd)
+	pingCmd.Flags().StringVarP(&pingWorkspace, "workspace", "w",
+		"", "Ping configuration with a specific workspace "+
+			"(Kong Enterprise only).\n"+
+			"Use this with an RBAC user.")
```

Comment:
`"Useful when RBAC permissions are scoped to a workspace"` might be more fitting.